pkg/security/controller/namespace_scc_allocation_controller.go

```diff
@@ -286,11 +295,14 @@ func (c *NamespaceSCCAllocationController) Repair(ctx context.Context, syncCtx f
 	if err != nil {
 		return err
 	}
+
+	namespacesProcessed := 0
 	for _, ns := range nsList {
 		value, ok := ns.Annotations[securityv1.UIDRangeAnnotation]
 		if !ok {
 			continue
 		}
+		namespacesProcessed++
 		block, err := uid.ParseBlock(value)
 		if err != nil {
```

Comment:
do we care about namespaces with invalid blocks? Should we publish these error in form of metrics or other form?

Reply:
It would be nice to at least warn about these, either in logs or through some kind of annotation. Can be done in a followup.

Reply:
+1, I will make a followup PR for it